src/osemgrep/cli_ci/Ci_subcommand.ml

```diff
@@ -302,10 +302,11 @@ let partition_findings ~keep_ignored (results : Out.cli_match list) =
 (*****************************************************************************)
 
 (* from rule_match.py *)
-let severity_to_int = function
-  | "EXPERIMENT" -> `Int 4
-  | "WARNING" -> `Int 1
-  | "ERROR" -> `Int 2
+let severity_to_int (severity : Out.severity) =
+  match severity with
+  | Experiment -> `Int 4
+  | Warning -> `Int 1
+  | Error -> `Int 2
   | _ -> `Int 0
```

Comment:
better be explicit and avoid _ (actually we have a semgrep rule against that; not sure why it didn't triggered here).